File: 1-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        for index1, m in enumerate(nums):
            for index2, n in enumerate(nums):
                if index2 <= index1:
                    continue
                if m + n == target:
                    return [index1, index2]

File: c/1-two-sum.c
/**
 * Note: The returned array must be malloced, assume caller calls free().
 */
int* twoSum(int* nums, int numsSize, int target, int* returnSize) {
    int* ret = (int*) malloc(sizeof(int) * 2);
    int i = 0;
    for (; i < numsSize; ++i) {
        for (int j = i+1; j < numsSize; ++j) {
            if (nums[i] + nums[j] == target) {
                ret[0] = i;
                ret[1] = j;
                *returnSize = 2;
                return ret;
            }
        }
    }
    *returnSize = 0;
    return malloc(0);
}

